bin/zField

```diff
@@ -150,8 +150,37 @@ if __name__ == '__main__':
             dMapDict = clusters.makeDensityMap(RADeg, decDeg, galaxyCatalog, z, dz, rMaxMpc = 1.5)
             astImages.saveFITS(args.outDir+os.path.sep+"densityMap_z%s_%.3f_%.3f.fits" % (z, RADeg, decDeg),
                                dMapDict['map'], dMapDict['wcs'])
-    
+            
+    #estimate errors for density map, CS and A
+    for montecarlo1 in range(0,1):
```

Comment:
This just runs one iteration, so don't use a for loop.

What you should add here is an if statement (e.g., `if args.estimateDensityMapErrors == True:`), after adding a command line switch to parser, so that estimating uncertainties is optional (as it takes a long time).